src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -0,0 +1,493 @@
+use std::{iter, num::NonZeroUsize};
```

Comment:
Since you separate the fold chip in a single file, can you add some comments in the beginning of file to explain roughly what does this chip do?